src/bitcoin-cli.cpp

```diff
@@ -291,6 +293,184 @@ class GetinfoRequestHandler: public BaseRequestHandler
     }
 };
 
+/** Process netinfo requests */
+class NetinfoRequestHandler : public BaseRequestHandler
+{
+private:
+    bool IsAddrIPv6(const std::string& addr) const
+    {
+        return addr.front() == '[';
```

Comment:
This is undefined behavior if `addr` is empty, you might want to guard against that.

Reply:
> This is undefined behavior if `addr` is empty, you might want to guard against that.

Thanks! will fix

Reply:
https://en.cppreference.com/w/cpp/string/basic_string/front "The behavior is undefined if empty() == true." Not sure how I missed that, good catch.